e2e/test/scenarios/native-filters/sql-field-filter-number.cy.spec.js

```diff
@@ -51,7 +51,6 @@ describe("scenarios > filters > sql filters > field filter > Number", () => {
 
         FieldFilter.setWidgetType(subType);
 
-        FieldFilter.openEntryForm({ isFilterRequired: true });
```

Comment:
This was removed because the `enterDefaultValue` helper has it's own click to open the modal. Clicking twice would actually cause the modal to close.